src/doc/src/commands/cargo-add.md

```diff
@@ -29,7 +29,7 @@ If no source is specified, then a best effort will be made to select one, includ
 * Workspace members
 * Latest release in the registry
 
-When you add a package that is already present, the existing entry will be updated with the flags specified.
+When you add a package that is already present, the existing entry will be updated with the [feature flags specified](#option-cargo-add---features).
```

Comment:
This is narrowing the scope of what is being communicated.  It will also update `--default-features`, `--optional`, the version, etc.

Reply:
Ah, so I misunderstood what this does. I'll remove this edit. I wonder if there is a clearer way to word this. Perhaps I was just being thick.

Reply:
Providing an example might help 

> the existing entry will be updated it the specified flags (e.g. [`--features`](...))